src/settings/ClosingReasons.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import ClosingReasonsForm from './ClosingReasonsForm';
+
+
+class ClosingReasons extends Component {
+  static manifest = Object.freeze({
+    recordId: {},
```

Comment:
you don't need `recordId` at all, you can reference lines limit setting as an example.